x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/tables/meta_engines_table_name_column_content.test.tsx

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mountWithIntl } from '../../../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiHealth } from '@elastic/eui';
+
+import { SchemaConflictFieldTypes, SchemaConflicts } from '../../../../../shared/types';
+import { EngineDetails } from '../../../engine/types';
+
+import { MetaEnginesTableNameColumnContent } from './meta_engines_table_name_column_content';
+
+describe('MetaEnginesTableNameColumnContent', () => {
+  it('includes the name of the engine', () => {
+    const wrapper = mountWithIntl(
```

Comment:
IMO, we shouldn't be using `mountWithIntl` for this component - it doesn't use any `<FormattedDate>` type components that would require `react-intl`, and more importantly it doesn't have a complex structure that would require a ton of diving/make `mount` preferrable, so we should stick to `shallow()` here.

If you really want to test the pluralization of the engine names, I would stick a `data-test-subj` on the EuiFlexItem parent and do a `.text()` or `.children()` to assert on its content, but I also don't think that's necessary, since we don't need to unit-test 3rd party pluralization (it's better tested in E2E tests if anywhere).

Reply:
Thanks for clarifying the use of `mountWithIntl` and how to check for text w/o it 😄